src/vs/workbench/parts/extensions/browser/extensionsActions.ts

```diff
@@ -544,6 +544,37 @@ export class EnableGloballyAction extends Action implements IExtensionAction {
 	}
 }
 
+export class IgnoreAction extends Action {
```

Comment:
`IgnoreExtensionRecommendationAction`

The name of the action should be self explanatory